MapboxDirections/MBCongestion.h

```diff
@@ -0,0 +1,12 @@
+typedef NS_OPTIONS(NSUInteger, MBCongestion) {
```

Comment:
A particular segment can’t have two levels at the same time, so we would want an `NS_ENUM` in Objective-C (`enum` in Swift), not an `NS_OPTIONS` in Objective-C (`OptionSet` in Swift).

Since normal enumerations bridge from Swift to Objective-C just fine, we can implement this as an ordinary Swift enumeration – no need for an Objective-C header.